data-prepper-plugins/s3-source/README.md

```diff
@@ -72,13 +72,13 @@ source-pipeline:
         region: "us-east-1"
         sts_role_arn: "arn:aws:iam::123456789012:role/Data-Prepper"
       scan:
-        start_time: 2023-01-21T18:00:00
-        end_time: 2023-04-21T18:00:00
+        start_time: now
```

Comment:
What does "now" mean? Is this the time Data Prepper starts? A regularly updated time?

Should we use a clearer name? Or a different field entirely?

Reply:
I did see that in the code. But, it seems a little unclear from the YAML. Why would "now" naturally mean start-up time? Why doesn't it mean what is says - "now?"

When I see the following, I'd think this is reading from "now" which is anytime that Data Prepper is running through 6 hours.

```
start_time: now
range: "PT6H"
```

Reply:
I can't think of any other keywords, do you have any ideas?

Reply:
Perhaps `start_time`?

Reply:
I'm going to merge this in for testing. Will get back to this before the release.